packages/features/calendars/weeklyview/components/event/Event.tsx

```diff
@@ -19,27 +49,32 @@ export function Event({
   onEventClick,
 }: EventProps) {
   const selected = currentlySelectedEventId === event.id;
+  const { options } = event;
+
+  const borderColor = options?.borderColor ? "custom" : options?.status;
+
+  const styles = options?.borderColor
+    ? {
+        borderColor: options?.borderColor,
+      }
+    : {};
```

Comment:
Use a "custom" border prop that applies 0 styling L:38 

Allowing us to use the dynamic styles